creusot/src/translation/builtins.rs

```diff
@@ -30,11 +30,17 @@ pub fn lookup_builtin(
         let r = args.remove(0);
 
         return Some(Exp::BinaryOp(BinOp::Sub, box l, box r));
-    } else if def_id == ctx.tcx.get_diagnostic_item(Symbol::intern("le_int")) {
```

Comment:
Don't forget to add support for `le` back!

Reply:
Yeah, I missed that. I have reverted the `le_int` case now.